inc/Cache/CacheRenderer.php

```diff
@@ -40,8 +40,10 @@ public function makeDefaultCacheDecision()
         // for wiki pages, check metadata dependencies
         $metadata = p_get_metadata($this->page);
 
-        if (!isset($metadata['relation']['references']) ||
-            empty($metadata['relation']['references'])) {
+        if (
+            !isset($metadata['relation']['references']) ||
+            empty($metadata['relation']['references'])
+        ) {
```

Comment:
I understand the logic of starting on a new line, but I would not write it this way myself. But probably it is a matter of taste.

Reply:
I actually like it...